src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -114,24 +113,24 @@ const RadioGroup = createClass({
 		// equal to true, then the index of the last one should override the
 		// value of the `selectedIndex` prop.
 		const actualSelectedIndex = selectedIndexFromChildren !== -1
-				? selectedIndexFromChildren
-				: selectedIndex;
+			? selectedIndexFromChildren
+			: selectedIndex;
 
 		return (
 			<span
-					{...passThroughs}
-					className={cx('&', className)}
+				{...passThroughs}
+				className={cx('&', className)}
 			>
 				{_.map(radioButtonChildProps, (radioButtonChildProp, index) => {
 					return (
 						<RadioButtonLabeled
-								{...radioButtonChildProp}
-								isSelected={actualSelectedIndex === index}
-								key={index}
-								callbackId={index}
-								name={name}
-								onSelect={this.handleSelected}
-								Label={_.get(_.first(findTypes(radioButtonChildProp, RadioGroup.Label)), 'props.children', null)}
+							{...radioButtonChildProp}
+							isSelected={actualSelectedIndex === index}
+							key={index}
+							callbackId={index}
+							name={name}
+							onSelect={this.handleSelected}
+							Label={_.get(_.first(findTypes(radioButtonChildProp, RadioGroup.Label)), 'props', null)}
```

Comment:
This is the actual fix. Most everything else was just whitespace changes, and tests.